frontend/javascripts/test/e2e-setup.ts

```diff
@@ -70,6 +71,13 @@ global.fetch = function fetchWrapper(url, options) {
     newUrl = `http://localhost:9000${url}`;
   }
 
+  if (!options) {
+    options = {};
```

Comment:
looks like this options thing was already used, e.g. in line 82. Why is this setup step needed now?